PluginExampleProject/src/main/scala/com/alpine/plugin/samples/ver1_0/ColumnFilterPlugin.scala

```diff
@@ -0,0 +1,195 @@
+package com.alpine.plugin.samples.ver1_0
+
+import com.alpine.plugin.core._
+import com.alpine.plugin.core.datasource.OperatorDataSourceManager
+import com.alpine.plugin.core.dialog.{ColumnFilter, OperatorDialog}
+import com.alpine.plugin.core.io._
+import com.alpine.plugin.core.spark.utils.SparkUtils
+import com.alpine.plugin.core.spark.{SparkIOTypedPluginJob, SparkRuntimeWithIOTypedJob}
+import org.apache.hadoop.fs.{FileSystem, Path}
+import org.apache.spark.SparkContext
+
+import scala.collection.mutable
+
+class ColumnFilterPlugin {
+
+}
+
+class ColumnFilterSignature extends OperatorSignature[
+  ColumnFilterGUINode,
+  ColumnFilterRuntime] {
+  def getMetadata(): OperatorMetadata = {
+    new OperatorMetadata(
+      name = "ColumnFilter (Spark)",
+      category = "Transformation",
+      author = "Egor Pakhomov",
+      version = 1,
+      helpURL = "",
+      iconNamePrefix = ""
+    )
+  }
+}
+
+
+class ColumnFilterGUINode extends
+OperatorGUINode[HdfsTabularDataset, HdfsTabularDataset] {
+  override def onPlacement(
+                            operatorDialog: OperatorDialog,
+                            operatorDataSourceManager: OperatorDataSourceManager,
+                            operatorSchemaManager: OperatorSchemaManager): Unit = {
+
+    operatorDialog.addTabularDatasetColumnCheckboxes(
+      "columnsToDelete",
```

Comment:
As previously discussed, columnsToKeep is more consistent with other operators that columnsToDelete.